app/views/admin_mailer/linkedin_weekly_profiles.html.erb

```diff
@@ -0,0 +1,26 @@
+<p>Here are the developer with linkedin profiles updated in past 7 days:</p>
+<table style="color: #6B7280; font-size: 0.875rem; line-height: 1.25rem; text-align: left; width: 100%; ">
+  <thead style="background-color: #F9FAFB; color: #374151; font-size: 0.75rem; line-height: 1rem; text-transform: uppercase; ">
+    <tr>
+      <th scope="col" style="padding-top: 0.75rem;padding-bottom: 0.75rem; padding-left: 1.5rem;padding-right: 1.5rem; ">Developer Name</th>
+      <th scope="col" style="padding-top: 0.75rem;padding-bottom: 0.75rem; padding-left: 1.5rem;padding-right: 1.5rem; ">Linkeidn URL</th>
+      <th scope="col" style="padding-top: 0.75rem;padding-bottom: 0.75rem; padding-left: 1.5rem;padding-right: 1.5rem; ">Current Company</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% @developer_external_profiles&.each do |developer_external_profile| %>
+    <tr style="background-color: #ffffff; border-bottom-width: 1px; ">
+      <th scope="row" style="padding-top: 1rem;padding-bottom: 1rem; padding-left: 1.5rem;padding-right: 1.5rem; color: #111827; font-weight: 500; white-space: nowrap; ">
+        <%= link_to developer_external_profile[:name], developer_url(developer_external_profile[:id]) %>                
```

Comment:
When I view the preview I see their "name" as the URL, too.

I also am thinking if it might make more sense to pass this as an object (like the comment above) instead of a hash. What do you think?

Reply:
Yeah I have added name as URL pointing to page of  developer profile in application so that in case you need to check any more details or contact developer through app it will be easy to do.

Have removed Hash and passing directly object to fetch details.